### DISCUSSION TITLE
how to use audio in cannoli

### DISCUSSION THREAD

**Discussion by winne617** [DeabLabs/cannoli#3] [Q&A] [answered]
hey there, exceptional idea and plugin! also big thanks for the extensive documentation in canvas format. devoured everything;)
only don't i know how to make use of audio. could you clarify that part when finding time? 
thx so much!

**Comment by blindmansion** [comment, marked as answer]
Thanks for trying it out!

If you're on desktop or android (transcription currently doesn't work on ios):
1. Make sure you have the audio recorder plugin enabled in "core plugins" settings
2. Enable them in settings ("Enable audio recorder triggered cannolis")
3. Go to a note with a cannoli property (and the value of the property is a link to a cannoli)
4. Record something
5. The recording will be transcribed and the transcription will replace the link to the audio file on that note
6. The cannoli will start

This is the current setup, trying to balance flexibility and not interrupting non-cannoli audio recordings. If you have suggestions or ideas for other ways you'd want to handle audio transcriptions in cannoli let me know!

Let me know if it still isn't working, I havent tested transcription everywhere yet.

**Reply by winne617** [reply]
thanks, just didn't activate the recorder plugin yet
